jet_fuel/app/models/url.rb

```diff
@@ -0,0 +1,18 @@
+class Url < ActiveRecord::Base
+  validates :url, presence: true, :format => {:with => URI.regexp}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.